.pipelines/cni/lsg/kernel-upgrade-template.yaml

```diff
@@ -0,0 +1,307 @@
+parameters:
+  dependsOn: ""
+  name: ""
+  clusterType: ""
+  clusterName: ""
+  nodeCount: ""
+  vmSize: ""
+  os: linux
+  arch: ""
+  osSKU: Ubuntu
+
+# CNIv2
+# + Should be able to scale up/down the pods successfully certain number of times.
+# + Node reboot scenarios should be covered.
+# + The CNS state should be validated with that of CNI state.
+# + Pods should have ips assigned and connectivity/datapath test should be present.
+# + CNS restart and validates the state
+
+stages:
+  - stage: create_${{ parameters.name }}
+    variables:
+      commitID: $[ stagedependencies.setup.env.outputs['SetEnvVars.commitID'] ]
+    pool:
+      name: $(BUILD_POOL_NAME_DEFAULT)
+    dependsOn:
+      - setup
+    displayName: "Create Cluster - ${{ parameters.clusterName }}"
+    jobs:
+      - job: create_aks_cluster_with_${{ parameters.name }}
```

Comment:
Does it something specific to kernel upgrade ? Wondering if we can use the template file we for creating the cluster.

Reply:
Will go ahead and make this an actual template.